docs/source/cli/model-training-inference/output-remapping.rst

```diff
@@ -1,6 +1,220 @@
-.. _output-remapping:
+.. _gs-output-remapping:
 
 GraphStorm Output Node ID Remapping
 ====================================
+During :ref:`Graph Construction<graph-construction>`, GraphStorm converts
+user provided node IDs into integer-based node IDs. Thus, the outputs of
+GraphStorm training and inference scripts, i.e., :ref:`saved node
+embeddings<gs-output-embs>` and :ref:`saved prediction results<gs-out-predictions>`,
+are stored with their integer-based node IDs. GraphStorm provides a
+``gconstruct.remap_result`` module to convert the integer-based node IDs
+into the original user provided node IDs according to the :ref:`node ID
+mapping files<gs-id-mapping-files>`.
 
-(Under Construction)
\ No newline at end of file
+.. note::
+
+    If the training or inference tasks are launched by GraphStorm CLIs,
+    the ``gconstruct.remap_result`` module is automatically triggered to
+    to remap the integer-based node IDs into the original user provided
+    node IDs.
+
+Output Node Embeddings
+----------------------
+By default, the output node embeddings after ``gconstruct.remap_result``
+are stored in parquet format. The node embeddings for different node
```

Comment:
in which location? May say they are stored in the same emb_dir as the input.

But if move the command and its explanation above this section, no need to mention the location here.

Reply:
Updated